doc/guide/build.hpp

```diff
@@ -104,6 +105,17 @@ In Ubuntu and Debian, you can get all of these dependencies through apt:
   libstb-dev
 @endcode
 
+@note For older versions of Ubuntu, Armadillo needs to be built from 
+source as apt installs an older version. So you need to omit 
+\c libarmadillo-dev from the code snippet above. Then 
+<a href="http://arma.sourceforge.net/download.html">this link</a>
+can be used to download the file and the README in the uncompressed folder
+can be followed to perform the build. <br> <br>
+Also note that if apt is not able to find \c libensmallen-dev and \c 
+libstb-dev, they are downloaded and installed automatically by 
+CMake if you keep the flags \c DOWNLOAD_ENSMALLEN and \c DOWNLOAD_STB_IMAGE 
```

Comment:
I think that the stb and enemallen cmake options are well described above (lines 83 and 87-88).

Reply:
Yeah my bad. I missed those few points during my read. Should i remove this?

Reply:
I added these as the snippet given was generating an error and many people like me would directly copy paste the snippet and might be confused. So i wrote the line with the exact problem that might occur.

Reply:
I think it's best to take them out; we also have #2235 open, which removes ensmallen and stb from this apt install in this same document.